python/lsst/ts/tunablelaser/component.py

```diff
@@ -121,9 +129,20 @@ async def set_output_energy_level(self, output_energy_level):
         self.log.debug(f"Changing output energy level={output_energy_level}")
         await self.m_cpu800.set_output_energy_level(output_energy_level)
 
-    async def set_burst_mode(self):
+    async def set_trigger_mode(self):
+        """Set trigger mode for laser."""
+        await self.m_cpu800.set_propagation_mode(Mode.TRIGGER)
+        await self.csc.publish_new_detailed_state(
+            LaserDetailedState.PROPAGATINGBURSTMODETRIGGERED
+        )
+        await self.csc.publish_new_detailed_state(
+            LaserDetailedState.PROPAGATINGBURSTMODEWAITINGFORTRIGGER
+        )
+
+    async def set_burst_mode(self, count):
```

Comment:
Thank you for consolidating this. At risk of making you wish you had not...

Please document the `count` parameter. What is the valid range?

What does this do if count is out of range (< 1 or too big). Does self.m_cpu800.set_burst_count catch this and raise a clear error? If not, please check it here and raise an error before calling self.m_cpu800.set_burst_count

Please document the exception in an Exceptions section. (I suppose any command that calls self.m_cpu800.x may raise if the laser is not connected or something. It seems excessive to document those exceptions here, but the exception for count out of range is less obvious.)